golem/managers/activity/mixins.py

```diff
@@ -10,6 +10,19 @@
 logger = logging.getLogger(__name__)
 
 
+class ActivityWrapperMixin:
```

Comment:
Looks like this class is not a mixin, but a base class.

Reply:
It is not as it has difrent `__init__`